### DISCUSSION TITLE
GitHub Copilot could not connect to server. Extension activation failed: "Unexpected end of JSON input"

### DISCUSSION THREAD

**Discussion by D4D25A** [community/community#19053] [Copilot Conversations]
This error shows. I have reinstalled the extension, VSCode, as well as disabled my antiviruses. Nothing seems to work. I do have authorization to use Copilot by the way, and I had just used it yesterday.

**Comment by Vladislav875** [comment]
Recently, such a problem also appeared when running Copilot on pycharm "Failed to initiate the GitHub login process. Please try again.", the same error as you have in VS Code

But it also happened after updating my Windows 11

Maybe the problem is on the GitHub side? Replacing the core file helped, but Copilot stopped working, even though I used it quite recently

**Comment by supermavster** [comment]
Hi @D4D25A and @Vladislav875 I found the solution here: [https://github.com/orgs/github-community/discussions/19061#discussioncomment-2995893](https://github.com/orgs/github-community/discussions/19061#discussioncomment-2995893)

**Comment by njoromyke** [comment]
Fuck you github.

**Comment by 201flaviosilva** [comment]
GitHub Copilot is now paid 🤑 : https://github.blog/2022-06-21-github-copilot-is-generally-available-to-all-developers/
